nss-day-cohort-48/giffygram-infoseek
# Have a gif to post?

#### Changes Made
1. Modifying the giffygram component was the only way I could get the button to display the way I wanted to
2. Added click event for the 'Have a gif to post?' button.
​
#### Steps to Review
1. Checkout this branch locally.
    ```
    git fetch --all
    git checkout pk-postlist
    serve
    ```
2. Open a new Terminal tab (⌘T) and navigate to the server directory.
3. Test app functionality.
    > Make sure the 'Have a gif to post?' button is working as expected
4. View code file.
    > Confirm file modifications are present as indicated above.
    > Confirm no unused code or extraneous comments exist.
